samples/previous-viewpoint/src/services/customService/CustomService.ts

```diff
@@ -0,0 +1,47 @@
+import { ServiceBase } from "@vertigis/web/services";
+import { command, MapsLike, ViewpointLike } from "@vertigis/web/messaging";
+
+export default class CustomService extends ServiceBase {
+    private _history: ViewpointLike[] = [];
+    private movedBecauseCommand = false;
```

Comment:
This one should also have a leading underscore since it's private.

Reply:
Updated.